File: src/main/java/dev/mazurkiewicz/quizer/QuizerApplication.java
package dev.mazurkiewicz.quizer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@ConfigurationPropertiesScan({"dev.mazurkiewicz.quizer.config"})
@SpringBootApplication
public class QuizerApplication {

	public static void main(String[] args) {
		SpringApplication.run(QuizerApplication.class, args);
	}

}

File: src/main/kotlin/pl/quizwiz/QuizerApplication.kt
package pl.quizwiz

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class QuizerApplication

fun main(args: Array<String>) {
    runApplication<QuizerApplication>(*args)
}
